sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/cosmos/properties/AzureCosmosProperties.java

```diff
@@ -14,16 +14,13 @@
 import com.azure.spring.cloud.autoconfigure.properties.core.proxy.HttpProxyCP;
 import com.azure.spring.service.cosmos.CosmosProperties;
 import org.springframework.boot.context.properties.NestedConfigurationProperty;
-import org.springframework.validation.annotation.Validated;
 
-import javax.validation.constraints.NotEmpty;
 import java.util.ArrayList;
 import java.util.List;
 
 /**
  * Configuration properties for Cosmos database, consistency, telemetry, connection, query metrics and diagnostics.
  */
-@Validated
```

Comment:
why remove this ?

Reply:
If @Validated is added to the configuruation properties and no validator is provided at runtime, it will throw exception. I took a look at the spring boot project, there's no @Validated on their configuration properties either.